### ISSUE TITLE
[Feature] - Add a feature to specify what level of WCAG compliance you require

### ISSUE THREAD

**Issue by MattMoranJava** [microsoft/accessibility-insights-web#1631]
## Is your feature request related to a problem? Please describe.
Currently my employer is auditing our website and upgrading accessibility to WCAG level A. We don't require AA or AAA. We're using Microsoft's Accessibility Insights tool as one method of auditing the site, but the tool only goes to level AA with no ability to change that. Consequently we have to keep a list of WCAG rules that are out of scope & wade through a bunch of false fails that we don't need to know about.
## Describe the desired outcome
If in the Settings there was (say) a slider that allowed you to select between levels of compliance, this would be very useful. The tool would only flag up the faults that are in scope for this accessibility upgrade.
## Describe alternatives you've considered
We might have to use other tools like Lighthouse.
## Additional context
No additional context

**Comment by ferBonnin** [comment]
Hi @MattMoranJava, thank you for the feature request! A quick question, do you use Automated checks or Assessment?

After an extensive chat, we came to the conclusion that we want to encourage our users to be WCAG 2.1 AA compliant, thus we do not want to add a filter for A only.
Also, we took a look and for the **automated checks** most of them are A, not AA.

**Comment by MattMoranJava** [comment]
Mostly automated at the moment, but the plan was to use the full spectrum
of tests available. Basically we have a pretty large website made up of a
lot of components, and the client for whom we maintain the website has
requested we upgrade to level A (for now - I hear that AA is going to be
required by law for any government sites soon, so it's likely that that
will go further, in the future). So our strategy so far has been to use
Lighthouse & Google to audit the accessibility, raise JIRA tickets based on
that, & now that we're halfway through the upgrade our managers have asked
us to do a final audit of the components that we've marked "done", using
the Microsoft Accessibility Insights browser extension. We intend to do a
first pass using the "FastPass" feature, and then go further into it with
the manual tools.

**Comment by ferBonnin** [comment]
Got it, thanks for the explanation for your scenario @MattMoranJava!

As mentioned we want to encourage our users to be WCAG 2.1 AA compliant but while you are in the journey of getting to AA, for the manual and assisted checks in Assessment, we add the WCAG Success Criterion that is covered in the _more info & examples_ panel.
Also, new WCAG 2.1 requirements are marked as such with a tag next to the requirement title.
